lib/marshalingCommands.js

```diff
@@ -0,0 +1,173 @@
+var _ = require('lodash');
+var unexpectedItemHandler = require('./unexpectedItemHandler');
+
+/**
+ * Converts boolean value to DynamoDb "BOOL"
+ *
+ * @param item
+ * @returns {*}
+ */
+function marshalBoolean(item) {
+  if (!_.isBoolean(item)) {
```

Comment:
This pattern is so common that maybe it could be pulled out into a separate function?

```js
const test = (predicate, item) => predicate ? item : void 0; 
```

I realise this implemenatation is contrary to the style guide but it's just an example and I'm lazy so I wanted to save key strokes.